API/NoDocs/Microsoft.AspNetCore.Builder/HttpLoggingApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,8 @@
+// Assembly 'Microsoft.AspNetCore.Diagnostics.Middleware'
+
+namespace Microsoft.AspNetCore.Builder;
+
+public static class HttpLoggingApplicationBuilderExtensions
+{
+    public static IApplicationBuilder UseHttpLoggingMiddleware(this IApplicationBuilder builder);
```

Comment:
Drop the Middleware suffix from method names. We don't do that anywhere (though, this is going away right?)

Reply:
Deleted https://github.com/dotnet/extensions/pull/4330/